ports/atmel-samd/common-hal/watchdog/WatchDogTimer.c

```diff
@@ -0,0 +1,312 @@
+/*
```

Comment:
The indentation in this file is all over the place, not sure what the standard should be.  In other files I more or less followed along.

Also, not sure how to handle attribution.  I'd like to link back to https://github.com/adafruit/Adafruit_SleepyDog/ if possible, considering it's probably a great resource to forward port for other arch if need be.

Reply:
I would say run it through `clang-format` or similar. @tannewt do we have a set of settings for clang-format that match what we do?

Just put a comment near the top that says it's derived from SleepyDog and link to the repo.

Reply:
I don't. I've been trying to ignore minor issues knowing I do want to run it through something. MicroPython settled on `astyle`: https://github.com/micropython/micropython/pull/5668 That's probably the simplest thing for us to adopt.

Reply:
See https://github.com/micropython/micropython/pull/5700. MicroPython adopted `uncrustify` instead. (I think it was the fourth formatter they tried)

Reply:
Done, but just the one file locally.  The upstream configs aren't in the tools/ dir here.